metaci/plan/templates/plan/detail.html

```diff
@@ -1,68 +1,98 @@
-{% block layout_header_text %}
+{% extends 'layout_full.html' %} {% block layout_header_text %}
 <div class="slds-page-header__row">
-    <div class="slds-text-body_regular">
-      Plan
-    </div>
-  </div>
-  {{ plan.name }}
-{% endblock %}
-
-{% block layout_header_details %}
+  <div class="slds-text-body_regular">Plan</div>
+</div>
+{{ plan.name }} {% endblock %} {% block layout_header_details %}
 <ul class="slds-grid slds-page-header__detail-row">
   <li class="slds-page-header__detail-block">
-    <p class="slds-text-title slds-truncate slds-m-bottom--xx-small" title="Role">Role</p>
+    <p
+      class="slds-text-title slds-truncate slds-m-bottom--xx-small"
+      title="Role"
+    >
+      Role
+    </p>
     <p class="slds-text-body--regular slds-truncate" title="{{ plan.role }}">
       {{ plan.role }}
     </p>
   </li>
   <li class="slds-page-header__detail-block">
-    <p class="slds-text-title slds-truncate slds-m-bottom--xx-small" title="Trigger">Trigger</p>
+    <p
+      class="slds-text-title slds-truncate slds-m-bottom--xx-small"
+      title="Trigger"
+    >
+      Trigger
+    </p>
     <p class="slds-text-body--regular slds-truncate" title="{{ plan.trigger }}">
       {{ plan.trigger }}
     </p>
   </li>
   <li class="slds-page-header__detail-block">
-    <p class="slds-text-title slds-truncate slds-m-bottom--xx-small" title="Regex">Regex</p>
+    <p
+      class="slds-text-title slds-truncate slds-m-bottom--xx-small"
+      title="Regex"
+    >
+      Regex
+    </p>
     <p class="slds-text-body--regular slds-truncate" title="{{ plan.regex }}">
       {{ plan.regex }}
     </p>
   </li>
   <li class="slds-page-header__detail-block">
-    <p class="slds-text-title slds-truncate slds-m-bottom--xx-small" title="Org">Org</p>
+    <p
+      class="slds-text-title slds-truncate slds-m-bottom--xx-small"
+      title="Org"
+    >
+      Org
+    </p>
     <p class="slds-text-body--regular slds-truncate" title="{{ plan.org }}">
       {{ plan.org }}
     </p>
   </li>
   <li class="slds-page-header__detail-block">
-    <p class="slds-text-title slds-truncate slds-m-bottom--xx-small" title="Flows">Flows</p>
+    <p
+      class="slds-text-title slds-truncate slds-m-bottom--xx-small"
+      title="Flows"
+    >
+      Flows
+    </p>
     <p class="slds-text-body--regular slds-truncate" title="{{ plan.flows }}">
       {{ plan.flows }}
     </p>
   </li>
   <li class="slds-page-header__detail-block">
-    <p class="slds-text-title slds-truncate slds-m-bottom--xx-small" title="Active?">Active?</p>
-    <p class="slds-text-body--regular slds-truncate slds-badge {% if plan.active %}slds-theme--success{% else %}slds-theme--error{% endif %}"
-      title="{{ build.get_status }}" title="{{ plan.active }}">
+    <p
+      class="slds-text-title slds-truncate slds-m-bottom--xx-small"
+      title="Active?"
+    >
+      Active?
+    </p>
+    <p
+      class="
+        slds-text-body--regular
+        slds-truncate slds-badge
+        {%
```

Comment:
The code formatter snarled this up good (they usually don't understand Django template tags). Let's revert all of the changes except for the `planrepo` swap.

Reply:
we could consider using a different extension (.html.tpl) for these so as not to provoke the formatter